cloud-operator/templates/rolebinding.yaml

```diff
@@ -26,4 +26,18 @@ subjects:
 roleRef:
   kind: Role
   name: hubble-client-certs-reader
-  apiGroup: rbac.authorization.k8s.io
\ No newline at end of file
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Only create this resource if the `gke-managed-dpv2-observability` namespace exists, i.e., if the chart is deployed into a GKE cluster:
```
{{- if (lookup "v1" "Namespace" "gke-managed-dpv2-observability" "") }}
...
{{- end }}
```